globus_jupyterlab/__init__.py

```diff
@@ -0,0 +1,16 @@
+
+import json
+import os.path as osp
+
+from ._version import __version__
+
+HERE = osp.abspath(osp.dirname(__file__))
+
+with open(osp.join(HERE, 'labextension', 'package.json')) as fid:
+    data = json.load(fid)
+
+def _jupyter_labextension_paths():
+    return [{
+        'src': 'labextension',
+        'dest': data['name']
+    }]
```

Comment:
The code here is all copied from the recommended Jupyterlab `__init__.py` extension code. This must have been optional when the package was created, and so we never included it.